misc/SW_WA_2021_2024/guilderton_to_geraldton/initial_stage/README.md

```diff
@@ -0,0 +1,12 @@
+# Polygons used to set the initial stage in the model
+
+The model points file `override_initial_stages.csv` from `model_multidomain_design_mod.f90`.
```

Comment:
Not sure if you should add "override_initial_stages.csv' here (or else a script that makes it?).